src/types/user.ts

```diff
@@ -66,6 +66,10 @@ export function toAppUser(from: SpidUser, token: string): User {
  */
 // tslint:disable-next-line:no-any
 export function validateSpidUser(value: any): Either<Error, SpidUser> {
+  if (!value.hasOwnProperty("fiscalNumber")) {
```

Comment:
I'm not super happy about this exception, we should probably change the constructor of `FiscalNumber` to accept the value returned by SPID as valid, and provide the international or local version of it... perhaps for another PR :)